<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ledit_db">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/carlos/dupfileinfo.sqlite3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>SQLite DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbut_loadPlaces">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load places</string>
    </property>
   </widget>
   <widget class="QListView" name="lview_places">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_newPlace">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/carlos/Documents</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbut_addPlace">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add place</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbut_scan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
